### PR TITLE
[strings] Fix messy formatting of countries_names.txt

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

File: tools/python/strings_utils.py [hunk 1]
```diff
@@ -345,24 +351,24 @@ def write_formatted(self, target_file=None, langs=None, keep_resolved=False):
                         before_block = "\n"
                     continue
 
-                outfile.write("{0}  {1}\n".format(before_block, key))
+                outfile.write("{0}{1}{2}\n".format(before_block, section_padding, key))
```
[the remark this answers is not in the document]
Comment (biodranik) [reply tools/python/strings_utils.py:354]: What is the standard format?

**Review by biodranik — APPROVED** [review]

File: tools/python/strings_utils.py [hunk 2]
```diff
@@ -332,6 +332,12 @@ def write_formatted(self, target_file=None, langs=None, keep_resolved=False):
 
         if target_file is None:
             target_file = self.strings_path
+        if target_file.endswith('countries_names.txt'):
+            section_padding = 0 * ""
```
Comment (pastk) [inline tools/python/strings_utils.py:336]:
```suggestion
            section_padding = 0 * " "
```
right?

**Review by pastk — APPROVED** [review]

File: tools/python/strings_utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pastk) [reply tools/python/strings_utils.py:354]: > What is the standard format?

The one used in other strings files
```
[[Types]]
  [type.highway.services]
    en = Service Area
    af = Diensgebied
```
No prefix for a section e.g. `[[...]]`.
Two spaces prefix for a `[...]` definition and four spaces prefix for a lang line.


I'm in favor of reformatting the `countries_names.txt` completely to have a single format.

**Comment by pastk** [comment]
Why the strings validation check fails though? :/

File: tools/python/strings_utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (biodranik) [reply tools/python/strings_utils.py:354]: The current countries_names.txt format is a valid subset of the official twine format. The file does not have any sections (and they are not needed).

[the remark this answers is not in the document]
Comment (pastk) [reply tools/python/strings_utils.py:354]: It is valid but requires a hack in the auto-formatter to use different indentation for different files, hence I argue its better to use the same indentation everywhere.

[the remark this answers is not in the document]
Comment (pastk) [reply tools/python/strings_utils.py:354]: e.g. the translations script will have to have a way to output slightly different indentation too, to support the current `countries_names.txt`

[the remark this answers is not in the document]
Comment (biodranik) [reply tools/python/strings_utils.py:354]: countries names is a temporary kostyl [hack], all countries, region names, and their translations should be generated from the underlying OSM data to be properly searched in all supported languages.

As countries modifications are rare, any manual indentation correction is feasible if necessary.

It doesn't make sense to spend much time on supporting something temporary.

[the remark this answers is not in the document]
Comment (pastk) [reply tools/python/strings_utils.py:354]: Exactly! That's why its better to use a "standard" indentation there and avoid temporary hacks in the tooling.

As you've said this file is rarely modified and manually referenced so a full re-format won't be an issue blame-history-wise.

**Comment by pastk** [comment]
The validation check is failing still...

**Pull request merged**